setup.cfg

```diff
@@ -24,8 +24,8 @@ xfail_strict=True
 filterwarnings=
     # Turn warnings that aren't filtered into exceptions
     error
-    # Deprecated GraphQL (including https://github.com/graphql-python/graphene/issues/1055)
-    ignore: GraphQLApp is deprecated and will be removed in a future release\..*:DeprecationWarning
+    # https://github.com/Tinche/aiofiles/issues/81
+    ignore: "@coroutine" decorator is deprecated.*:DeprecationWarning
```

Comment:
Is this still a thing? I thought we got rid of our dependency on aiofiles when we moved to AnyIO.

Reply:
This is probably the result of a bad rebase strategy. :sweat_smile: 

I've removed and double-checked everything else :+1: 

Thanks!